src/docs/Dependency injection.md

```diff
@@ -75,10 +75,10 @@ guideline.
 
 Currently the following Akka.NET Dependency Injection plugins are available:
 
-## AutoFac
+## Autofac
 
 In order to use this plugin, install the Nuget package with
-`Install-Package Akka.DI.AutoFac`, then follow the instructions:
+`Install-Package Akka.DI.Autofac`, then follow the instructions:
```

Comment:
While the name is indeed Autofac, our nuget package is still Akka.DI.AutoFac.
If you wish to correct the package name, here is the repo for it
https://github.com/akkadotnet/Akka.DI.AutoFac . Until that is resolved, I suggest we keep the install instructions like they are for now.